skimage/restoration/_denoise.py

```diff
@@ -355,3 +361,98 @@ def denoise_tv_chambolle(im, weight=50, eps=2.e-4, n_iter_max=200,
         raise ValueError('only 2-d and 3-d images may be denoised with this '
                          'function')
     return out
+
+
+def _sigma_est_dwt(detail_coeffs, distribution='Gaussian'):
+    """
+    Calculation of the robust median estimator of the noise standard
+    deviation.
+
+    Parameters
+    ----------
+    detail_coeffs : ndarray
+        The detail coefficients corresponding to the discrete wavelet
+        transform of an image.
+    distribution : str
+        The underlying noise distribution.
+
+    Returns
+    -------
+    sigma : float
+        estimated noise standard deviation.  See [1]_
+
+    References
+    ----------
+    .. [1] Donoho, David L., and Jain M. Johnstone. "Ideal spatial adaptation
+       by wavelet shrinkage." Biometrika 81.3 (1994): 425-455.
+    """
+    detail_coeffs = np.asarray(detail_coeffs)
```

Comment:
Is the call to `asarray` a standard thing in skimage, even when input is defined to be an array in the docstring?